--- deadbeef-0.7.2/plugins/gtkui/Makefile.am.orig	2018-02-16 11:15:14.524949113 +0000
+++ deadbeef-0.7.2/plugins/gtkui/Makefile.am	2018-02-16 11:44:43.595322829 +0000
@@ -1,9 +1,5 @@
 EXTRA_DIST = deadbeef.glade
 
-if OS_OSX
-osx_sources = retina.m
-endif
-
 GTKUI_SOURCES =    gtkui.c gtkui.h\
 				   callbacks.c interface.c support.c callbacks.h interface.h support.h\
 				   ddblistview.c ddblistview.h\
@@ -79,14 +75,13 @@
 endif
 
 if HAVE_GTK3
-GTKUI_SOURCES_GTK3 = $(GTKUI_SOURCES) $(osx_sources)
+GTKUI_SOURCES_GTK3 = $(GTKUI_SOURCES)
 
 ddb_gui_GTK3_la_LDFLAGS = -module -avoid-version
 
 ddb_gui_GTK3_la_SOURCES = $(GTKUI_SOURCES_GTK3)
 ddb_gui_GTK3_la_LIBADD = $(LDADD) $(GTK3_DEPS_LIBS) $(SM_LIBADD) ../libparser/libparser.a $(JANSSON_LIBS)
 ddb_gui_GTK3_la_CFLAGS = -std=c99 $(GTK3_DEPS_CFLAGS) $(SM_CFLAGS) $(JANSSON_CFLAGS)
-ddb_gui_GTK3_la_OBJCFLAGS = $(GTK3_DEPS_CFLAGS) $(SM_CFLAGS) $(JANSSON_CFLAGS)
 
 endif
 
